go/vt/vttablet/onlineddl/vrepl/types.go

```diff
@@ -92,6 +94,11 @@ func (c *Column) IsNumeric() bool {
 	return c.NumericPrecision > 0
 }
 
+// IsIntegral returns true if the column is some form of an integer
+func (c *Column) IsIntegral() bool {
```

Comment:
Nit, but IMO `IsIntegralType()` would be good here as I think it's more explicit/descriptive and then the name used is uniform between the two packages.

Reply:
Renamed